tests/sentry/api/endpoints/test_event_attachments.py

```diff
@@ -12,23 +13,24 @@ class EventAttachmentsTest(APITestCase):
     def test_simple(self):
         self.login_as(user=self.user)
 
-        project = self.create_project()
-
-        release = self.create_release(project, self.user)
-
-        group = self.create_group(project=project, first_release=release)
-
-        event1 = self.create_event(
-            event_id='a',
-            group=group,
-            datetime=datetime(2016, 8, 13, 3, 8, 25),
-            tags={'sentry:release': release.version}
+        min_ago = (timezone.now() - timedelta(minutes=1)).isoformat()[:19]
+        event1 = self.store_event(
+            data={
+                # 'event_id': 'a' * 32,
+                'fingerprint': ['group1'],
+                'timestamp': min_ago,
+                # 'tags': {'sentry:release': release.version},
```

Comment:
More commented code that could be removed.

Reply:
weird, I thought I removed all of them. Will fix